src/views/explore/explore.jsx

```diff
@@ -100,11 +115,17 @@ var Explore = injectIntl(React.createClass({
     },
     getTab: function (type) {
         var classes = classNames({
-            active: (this.props.itemType === type)
+            active: (this.props.itemType === type),
+            inactive: (this.props.itemType !== type)
```

Comment:
If it can't be both, you might be able to get away with just having `active` be present or not, and having the `hover` styling for `inactive` just be the default hover styling, perhaps.